src/utils/checkNavLinks.ts

```diff
@@ -0,0 +1,21 @@
+import { NavLink } from "../types";
+
+interface Props {
+  to?: string
+  items?: NavLink[]
+  pathCheck: string
+  tracker?: boolean
+}
+export const checkNavLinks = ({ to, items, pathCheck, tracker = false }: Props): boolean => {
```

Comment:
Why is `tracker` a param here? Not being used on function call. IMO makes more sense to define it as a let var inside the function (also a boolean param can't be reassigned later like in the conditionals below I think).

Reply:
Good catch, thanks @nhsz... patched!